integration_tests/test_employment.py

```diff
@@ -73,3 +75,153 @@ def test_movers_change_employment(populations):
         assert (
             before[should_change]["employer_id"] != after[should_change]["employer_id"]
         ).all()
+
+
+def test_employment_income_propensity_updates(tracked_live_populations):
+    for time_step in range(max(TIME_STEPS_TO_TEST)):
+        (
+            employer_ids_before,
+            employer_propensity_before,
+            employer_ids_after,
+            employer_propensity_after,
+        ) = get_consecutive_time_step_employer_ids_and_test_column_for_comparable_sims(
+            tracked_live_populations, time_step, "employer_income_propensity"
+        )
+
+        # Find those that change employers
+        changed_jobs_idx = employer_ids_before.index[
+            employer_ids_before != employer_ids_after
+        ]
+
+        assert (
+            employer_propensity_before.loc[changed_jobs_idx]
+            != employer_propensity_after.loc[changed_jobs_idx]
+        ).all()
+
+
+def test_no_employment_income_propensity_updates(tracked_live_populations):
```

Comment:
I think these two tests could become one test pretty easily by making `changed_jobs` and `changed_employer_propensity` booleans and asserting that they are always equal.